### DISCUSSION TITLE
How to enable ZTS?

### DISCUSSION THREAD

**Discussion by kyletaylored** [shivammathur/homebrew-php#1671] [Q&A / Help]
I'm trying to use the Parallel extension, which isn't part of [homebrew-extensions](https://github.com/shivammathur/homebrew-extensions), but the minimum requirement is to have ZTS enabled.

```
checking for ZTS... configure: error: parallel requires ZTS, please use PHP with ZTS enabled
```

Is there a way to enable thread safety when installing via homebrew, or do I need to manually compile it?

**Comment by shivammathur** [comment]
This tap does not have `zts` builds yet. You will have to compile it yourself. you can edit the formula file and build from source.
